CONTRIBUTING.md

```diff
@@ -85,25 +85,28 @@ development with the following:
 
   For Url:
 
-  `http://localhost:5000/`
+  `http://localhost:5000/auth/myusa/callback`
```

Comment:
contrary to what I have in my own app setup, this URL does not actually need have this value. Previous value was fine

Sorry for not clarifying that earlier (that being said, having this value doesn't break system, but it isn't necessary)